doc/classes/XRFaceTracker.xml

```diff
@@ -0,0 +1,469 @@
+			Raises both cheeks.
+		</constant>
+		<constant name="FT_LIP_SUCK_UPPER" value="119" enum="BlendShapeEntry">
+			Tucks in the upper lips.
+		</constant>
+		<constant name="FT_LIP_SUCK_LOWER" value="120" enum="BlendShapeEntry">
+			Tucks in the lower lips.
+		</constant>
+		<constant name="FT_LIP_SUCK" value="121" enum="BlendShapeEntry">
+			Tucks in both lips.
+		</constant>
+		<constant name="FT_LIP_FUNNEL_UPPER" value="122" enum="BlendShapeEntry">
+			Funnels in the upper lips.
+		</constant>
+		<constant name="FT_LIP_FUNNEL_LOWER" value="123" enum="BlendShapeEntry">
+			Funnels in the lower lips.
+		</constant>
+		<constant name="FT_LIP_FUNNEL" value="124" enum="BlendShapeEntry">
+			Funnels in both lips.
+		</constant>
+		<constant name="FT_LIP_PUCKER_UPPER" value="125" enum="BlendShapeEntry">
+			Upper lip part pushes outwards.
+		</constant>
+		<constant name="FT_LIP_PUCKER_LOWER" value="126" enum="BlendShapeEntry">
+			Lower lip part pushes outwards.
+		</constant>
+		<constant name="FT_LIP_PUCKER" value="127" enum="BlendShapeEntry">
+			Lips push outwards.
+		</constant>
+		<constant name="FT_MOUTH_UPPER_UP" value="128" enum="BlendShapeEntry">
+			Raises the upper lips.
+		</constant>
+		<constant name="FT_MOUTH_LOWER_DOWN" value="129" enum="BlendShapeEntry">
+			Lowers the lower lips.
+		</constant>
+		<constant name="FT_MOUTH_OPEN" value="130" enum="BlendShapeEntry">
+			Mouth opens, revealing teeth.
+		</constant>
+		<constant name="FT_MOUTH_RIGHT" value="131" enum="BlendShapeEntry">
+			Moves mouth right.
+		</constant>
+		<constant name="FT_MOUTH_LEFT" value="132" enum="BlendShapeEntry">
+			Moves mouth left.
+		</constant>
+		<constant name="FT_MOUTH_SMILE_RIGHT" value="133" enum="BlendShapeEntry">
+			Right side of the mouth smiles.
+		</constant>
+		<constant name="FT_MOUTH_SMILE_LEFT" value="134" enum="BlendShapeEntry">
+			Left side of the mouth smiles.
+		</constant>
+		<constant name="FT_MOUTH_SMILE" value="135" enum="BlendShapeEntry">
+			Mouth expresses a smile.
+		</constant>
+		<constant name="FT_MOUTH_SAD_RIGHT" value="136" enum="BlendShapeEntry">
+			Right side of the mouth expresses sadness.
+		</constant>
+		<constant name="FT_MOUTH_SAD_LEFT" value="137" enum="BlendShapeEntry">
+			Left side of the mouth expresses sadness.
+		</constant>
+		<constant name="FT_MOUTH_SAD" value="138" enum="BlendShapeEntry">
+			Mouth expresses sadness.
+		</constant>
+		<constant name="FT_MOUTH_STRETCH" value="139" enum="BlendShapeEntry">
+			Mouth stretches.
+		</constant>
+		<constant name="FT_MOUTH_DIMPLE" value="140" enum="BlendShapeEntry">
+			Lip corners dimple.
+		</constant>
+		<constant name="FT_MOUTH_TIGHTENER" value="141" enum="BlendShapeEntry">
+			Mouth tightens.
+		</constant>
+		<constant name="FT_MOUTH_PRESS" value="142" enum="BlendShapeEntry">
+			Mouth presses together.
+		</constant>
+		<constant name="FT_MAX" value="143" enum="BlendShapeEntry">
+			Maximum blend shape.
```

Comment:
I am to assume it means this, which is a standardized message across the class reference:
```suggestion
			Represents the size of the [enum BlendShapeEntry] enum.
```

Reply:
Done.